src/shared/fstab-util.c

```diff
@@ -34,6 +34,21 @@
 #include "strv.h"
 #include "util.h"
 
+bool fstab_has_fstype(const char *fstype) {
```

Comment:
not that it matters much, but I much prefer functions that don't eat up errors needlessly. Specifically, in this case I think the return value should be an int, and return negative errno if we encounter some serious issues, and 0 and 1 as "clean" replies. That way the caller can decide what to do with failures, whether to consider them or ignore them.